lib/nuntiux.ex

```diff
@@ -114,12 +114,25 @@ defmodule Nuntiux do
   """
   @spec history(process_name) :: ok | error
         when process_name: process_name(),
-             ok: [event()],
+             ok: Nuntiux.Mocker.history(),
```

Comment:
I'll express this type in this module, otherwise there's a roundtrip to understand what it is...